api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1841,6 +1841,62 @@ message CreateAuthenticateChallengeRequest {
   // If you are issuing challenges from the root cluster, but accessing a leaf,
   // call [AuthService.IsMFARequired] in the leaf instead of setting this field.
   IsMFARequiredRequest MFARequiredCheck = 5 [(gogoproto.jsontag) = "mfa_required_check,omitempty"];
+  // Extensions are extensions that will be apply to the issued MFA challenge.
+  // Extensions only apply to webauthn challenges currently. This field is required.
+  MFAChallengeExtensions Extensions = 6 [(gogoproto.jsontag) = "extensions,omitempty"];
```

Comment:
```suggestion
  MFAChallengeExtensions ChallengeExtensions = 6 [(gogoproto.jsontag) = "challenge_extensions,omitempty"];
```

Reply:
As commented in other PRs, being more specific to avoid confusing this with "regular" WebAuthn extensions.